src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -247,8 +250,18 @@ private void navigateToNextEntry() {
         entryEditorTabs.add(new OtherFieldsTab(databaseContext, libraryTab.getSuggestionProviders(), undoManager, dialogService, preferencesService, stateManager, Globals.entryTypesManager, ExternalFileTypes.getInstance(), Globals.TASK_EXECUTOR, Globals.journalAbbreviationRepository));
 
         // General fields from preferences
+        int i = 0;
         for (Map.Entry<String, Set<Field>> tab : entryEditorPreferences.getEntryEditorTabList().entrySet()) {
-            entryEditorTabs.add(new UserDefinedFieldsTab(tab.getKey(), tab.getValue(), databaseContext, libraryTab.getSuggestionProviders(), undoManager, dialogService, preferencesService, stateManager, Globals.entryTypesManager, ExternalFileTypes.getInstance(), Globals.TASK_EXECUTOR, Globals.journalAbbreviationRepository));
+            String key = tab.getKey();
+            if (key.equals(preferencesService.getDefaults().get(JabRefPreferences.CUSTOM_TAB_NAME + "_def" + i))) {
+                key = Localization.lang(key);
+            }
+            Set<Field> fields = tab.getValue();
+            if (fields.equals(FieldFactory.parseFieldList((String) Objects.requireNonNullElse(preferencesService.getDefaults().get(JabRefPreferences.CUSTOM_TAB_FIELDS + "_def" + i), "")))) {
+                fields = fields.stream().map(Field::getName).map(Localization::lang).map(FieldFactory::parseField).collect(Collectors.toSet());
```

Comment:
I'm not sure if that really works as intended, or if it now also stores the value typed in the field under the localized field name. But maybe we shouldn't localize field names at all or be consistently and localize all field names, also in other tabs.

Reply:
The idea is to store pref X only as X if it was changed by the user. Otherwise store the default value.
This is needed because JabRef writes all preferences once it's closed. Otherwise, if we localize 'Abstract' to 'Zusammenfassung' it would be stored as 'Zusammenfassung' and cannot be localized again. So no matter what language I set, JabRef would always say 'Zusammenfassung'.

Reply:
No the value should not be recorded in the preferences, only the heading of the tab and the fields to display in that tab. Those strings can be changed in the preferences, not directly in the entry editor AFAIK.

Reply:
I meant not the preferences, but how the field is written in the Bib file